ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -28,6 +28,10 @@ images:
   dockerfile_path: build/create-vm/Dockerfile
   from: openshift_release_golang-1.15
   to: tekton-task-create-vm
+- context_dir: modules/copy-template/
+  dockerfile_path: build/copy-template/Dockerfile
+  from: openshift_release_golang-1.15
```

Comment:
@ksimon1 it would be great if you could make a PR updating these  (from) due to https://github.com/kubevirt/kubevirt-tekton-tasks/pull/95 being merged